pages/index.js

```diff
@@ -13,7 +13,10 @@ export default function Home() {
       <HomeContainer>
         <InfoContainer>
           <h3>Welcome to RPG Sheet-Creator!</h3>
-          <p>Create: create your Character-Sheet.</p>
+          <p>
+            Create: create your Character-Sheet. <br />
+            (Images should be square!)
```

Comment:
Users be wildin'. Perhaps it would be beneficial to account for them uploading any type of format. But for the time being, this'll do. :)